orchestrator/redisManager.js

```diff
@@ -0,0 +1,22 @@
+/* jslint node: true */
+"use strict";
+
+var redis = require("redis");
+var config = require("./config");
+var { ClientWrapper } = require('./redisClientWrapper');
+
+class RedisManager {
+  constructor() {
+    this.redis = redis.createClient({ host: config.redis.url });
```

Comment:
I think Redis export some events, it could be nice to debug. See the [documentation](https://www.npmjs.com/package/redis) for more details.

Reply:
Thanks